src/components/TransactionConfirmationModal/index.tsx

```diff
@@ -63,16 +63,34 @@ export function ConfirmationPendingContent({ onDismiss, pendingText }: { onDismi
 }
 
 function TransactionSubmittedContent({
+  trade,
   onDismiss,
   chainId,
   hash,
 }: {
+  trade?: Trade
   onDismiss: () => void
   hash: string | undefined
   chainId: ChainId
 }) {
   const theme = useTheme()
 
+  const isGnosisProtocolTrade = trade instanceof GnosisProtocolTrade
+
+  const externalLink = chainId && hash && (
```

Comment:
Same. I think it should be capitalize because it's a component.

Something like `ExplorarExternalLink` (just an idea)

Reply:
I disagree with the capitalization. I searched for a couple of standards and didn't find.
`ExplorerExternalLink`
^
on a local variable declaration. I'm willing to debate this if you're up for it.
I did made the name more specific.

Reply:
To make sense it would have to be outside of this component (`TransactionSubmittedContent`). 
But I'm fine with leaving it here, as we don't have a clear pattern.